### ISSUE TITLE
the leaftleft.draw.js provided in the git causes "L.Control.Draw is not a constructor" error

### ISSUE THREAD

**Issue by aleifuu** [Leaflet/Leaflet.draw#831]
- [ ] I'm reporting a bug, not asking for help
- [ ] I've looked at the [documentation](http://leaflet.github.io/Leaflet.draw/dist/reference-0.4.1.html) to make sure the behaviour is documented and expected
- [ ] I'm sure this is a Leaflet Draw code issue, not an issue with my own code nor with the framework I'm using (Cordova, Ionic, Angular, React…)
- [ ] I've searched through the issues to make sure it's not yet reported

----

## How to reproduce

- Leaflet version I'm using: Leaflet 1.3.1+ 
- Leaflet Draw version I'm using: Leaflet.draw.js 0.4.2
- Browser (with version) I'm using: 63.0.3239.140 (Official Build) (64-bit)
- OS/Platform (with version) I'm using: ChromeOS

just load the following on your <head> section

<script src=<?php echo base_url();?>assets/js/Leaflet.Draw/leaflet.draw.js></script>
<script src=<?php echo base_url();?>assets/js/Leaflet.Draw/Leaflet.Draw.Event.js></script>
<link rel="stylesheet" href=<?php echo base_url();?>assets/js/Leaflet.Draw/leaflet.draw.css>

and in <script> 
var drawnItems = new L.FeatureGroup();
mymap.addLayer(drawnItems);

var drawControl = new L.Control.Draw({
    edit: {
        	featureGroup: drawnItems
          }
     });
mymap.addControl(drawControl);

When your page loads, and using Inspect. it will return L.Control.Draw is not a constructor error. Of course, it broke your page. I was using leaflet to draw salesman's journey and I had to comment out this part 'var drawControl = new L.Control.Draw' to make sure page loads fine again

## Minimal example reproducing the issue

- [ ] this example is as simple as possible
- [ ] this example does not rely on any third party code

## How to fix ?
I had to use Google's one for this to work 
https://cdnjs.cloudflare.com/ajax/libs/leaflet.draw/0.4.2/leaflet.draw.js

basically replace the git's version to that one, then it's working

**Comment by benjaminhoffman** [comment]
im getting the same error.  i want to use a bundler (ie, webpack) so dont really want to use the CDN.

@aleifuu did you find an alternative solution?

**Comment by benjaminhoffman** [comment]
hopefully this ticket helps address this issue.

https://github.com/Leaflet/Leaflet.draw/issues/880
